src/paperless/settings.py

```diff
@@ -92,6 +94,13 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+# We allow CORS from localhost:8080
+CORS_ORIGIN_WHITELIST = ("localhost:8080")
+_allowed_cors_hosts = os.getenv("PAPERLESS_CORS_ALLOWED_HOSTS")
+if _allowed_cors_hosts:
+    CORS_ORIGIN_WHITELIST = tuple(_allowed_cors_hosts.split(","))
```

Comment:
All the 4 lines are the same as... :
`CORS_ORIGIN_WHITELIST = os.getenv("PAPERLESS_CORS_ALLOWED_HOSTS", "localhost:8080").split(",")`

Reply:
I followed the code pattern of ALLOWED_HOSTS as I didn't want to introduce a different style of writing.  WDYT?

Reply:
I like @sbrunner's suggestion, as it's more conscise.  I see what you mean about the ALLOWED_HOSTS though.  We can fix that later.  For now, having this in one statement is better, so let's go with that.